cluster/ubuntu/util.sh

```diff
@@ -263,11 +265,11 @@ EOF
 # Vars set:
 #   KUBE_MASTER
 #   KUBE_MASTER_IP
-function detect-master {
+function detect-master() {
   source "${KUBE_ROOT}/cluster/ubuntu/${KUBE_CONFIG_FILE-"config-default.sh"}"
   setClusterInfo
-  KUBE_MASTER=$MASTER
-  KUBE_MASTER_IP=$MASTER_IP
+  KUBE_MASTER=${MASTER}
```

Comment:
Note the lack of quoting around the variable references on the right hand side.  This is not consistent with other parts of this script, which attempt triply-nested quoting in order to quote the same variable references.

Is the quoting needed?  If not, then how about changing the script to never do it?
If you are unsure whether the quoting is needed, then you should quote every reference to the same variable.